sdk/textanalytics/azure-ai-textanalytics/src/test/java/com/azure/ai/textanalytics/TextAnalyticsClientBuilderTest.java

```diff
@@ -45,7 +47,8 @@
     /**
      * Test client builder with valid API key
      */
-    @Test
+    @ParameterizedTest(name = DISPLAY_NAME_WITH_ARGUMENTS)
```

Comment:
just curious why call "display name with arguments" since there is no argument at the method input.

Reply:
I am supposed to have a draft here. The PR is in progress.